primers.go

```diff
@@ -103,3 +104,79 @@ func MeltingTemp(sequence string) float64 {
 	meltingTemp, _, _ := SantaLucia(sequence, primerConcentration, saltConcentration, magnesiumConcentration)
 	return meltingTemp
 }
+
+/******************************************************************************
+Start of the De Bruijn stuff
+******************************************************************************/
+
+// DeBruijn generates a DNA DeBruijn sequence
+// https://rosettacode.org/wiki/De_Bruijn_sequences#Go
+// Pulled and adapted from here
+func DeBruijn(n int) string {
+	k := 4
+	alphabet := "ATGC"
+	a := make([]byte, k*n)
+	var seq []byte
+	var db func(int, int) // recursive closure
+	db = func(t, p int) {
```

Comment:
Can you change all single letter variables to whole words that are readable?

Reply:
I would, but I don't really know the proper names for all of them or precisely what they do

Reply:
Can you or someone find out?

Reply:
@Koeng101 still got single letter variables. Have no idea what's going on.

Reply:
single letter variables are also really tough to grep and refactor.

Reply:
Moving conversation down.